### PR TITLE
Add umoci test for Kubic

### PR DESCRIPTION
**Pull request by drpaneas**
- Related ticket: -
- Needles: -
- Verification run: http://skyrim.qam.suse.de/tests/2615#step/umoci/44

### CODE REVIEW

File: products/caasp/main.pm [hunk 1]
```diff
@@ -124,7 +124,9 @@ sub load_feature_tests {
     if (!check_var('SYSTEM_ROLE', 'microos')) {
         loadtest 'console/docker';
         loadtest 'console/docker_runc';
+        # OCI Containers
         loadtest 'console/skopeo' if is_caasp('kubic') && check_var('SYSTEM_ROLE', 'plain');
+        loadtest 'console/umoci'  if is_caasp('kubic') && check_var('SYSTEM_ROLE', 'plain');
```
Comment (mitiao) [inline products/caasp/main.pm]: It's duplicate of `is_caasp('kubic') && check_var('SYSTEM_ROLE', 'plain')`

File: tests/console/umoci.pm [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+sub run {
+    select_console("root-console");
+
+    record_info 'Test #1', 'Test: Installation';
+    my $package = 'umoci';
+    trup_install($package);
+
+    record_info 'Setup', 'Requirements';
+    assert_script_run('mkdir test_umoci && cd test_umoci');
+    assert_script_run('skopeo copy docker://docker.io/opensuse/tumbleweed oci:opensuse:tumbleweed', 300);
```
Comment (kravciak) [inline tests/console/umoci.pm]: Please increase timeout to 600 - we had some failures yesterday with 300 - see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4676

File: tests/console/umoci.pm [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    assert_script_run("ls -l bundle | grep 'config.json\\|rootfs\\|sha256\\|umoci.json'");
+    assert_script_run("cat bundle/rootfs/etc/os-release | grep 'Tumbleweed'");
+
+    record_info 'Test #4', 'Test: Unpacking a rootless image';
+    assert_script_run('umoci unpack --rootless --image opensuse:tumbleweed bundle2');
+
+    record_info 'Test #5', 'Test: Repacking both images';
+    assert_script_run('umoci repack --image opensuse:new bundle');
+    assert_script_run('umoci repack --image opensuse:new bundle2');    # rootless image
+}
```
Comment (kravciak) [inline tests/console/umoci.pm]: Please add `script_run "cd"` at the end to leave it clean for following tests

File: tests/console/umoci.pm [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    assert_script_run('mkdir test_umoci && cd test_umoci');
+    assert_script_run('skopeo copy docker://docker.io/opensuse/tumbleweed oci:opensuse:tumbleweed', 300);
+
+    record_info 'Test #2', 'Test: Creating an image';
+    assert_script_run('umoci init --layout new_image');          # without tag
+    assert_script_run('umoci new --image new_image:new_tag');    # with tag
+
+    record_info 'Test #3', 'Test: Unpacking an image';
+    assert_script_run('umoci unpack --image opensuse:tumbleweed bundle');
+    assert_script_run("ls -l bundle | grep 'config.json\\|rootfs\\|sha256\\|umoci.json'");
```
Comment (kravciak) [inline tests/console/umoci.pm]: If you use egrep you don't have to escape pipes - but that's up to you, functionality is the same :)

File: tests/console/umoci.pm [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+use base "consoletest";
+use testapi;
+use caasp;
+
+sub run {
+    select_console("root-console");
+
+    record_info 'Test #1', 'Test: Installation';
+    my $package = 'umoci';
+    trup_install($package);
```
Comment (kravciak) [inline tests/console/umoci.pm]: Why do you define $package variable and not just call `trup_install('umoci')` - but if you prefer this way feel free to ignore this comment!

**Comment by drpaneas** [comment]
I made the changes. Thanks for the tips and the advices once again :)

**Pull request merged**